code/game/gamemodes/cult/talisman.dm

```diff
@@ -225,16 +225,18 @@
 
 /obj/item/paper/talisman/horror/afterattack(mob/living/target, mob/living/user)
 	if(iscultist(user) && (get_dist(user, target) < 7))
-		to_chat(user, "<span class='cultitalic'>You disturb [target] with visions of madness!</span>")
 		if(iscarbon(target))
+			to_chat(user, "<span class='cultitalic'>You disturb [target] with visions of madness!</span>")
 			var/mob/living/carbon/H = target
 			H.reagents.add_reagent("mindbreaker", 12)
 			if(is_servant_of_ratvar(target))
 				to_chat(target, "<span class='userdanger'>You see a brief but horrible vision of Ratvar, rusted and scrapped, being torn apart.</span>")
 				target.emote("scream")
 				target.confused = max(0, target.confused + 3)
 				target.flash_act()
-		qdel(src)
+			qdel(src)
+		else
+			to_chat(user, "<span class='cultitalic'>You can't use \the [src] on that!</span>")
```

Comment:
This is going to be annoying getting this message every time you try to melee a mob and miss